### DISCUSSION TITLE
Hi, is RTL (Right to left) supported in this UI lib?

### DISCUSSION THREAD

**Discussion by jcyuan** [gluestack/gluestack-ui#2546] [Q&A] [answered]
thanks.

**Comment by Viraj-10** [comment, marked as answer]
Hey @jcyuan, It is supported by default. Please check rn [docs](https://reactnative.dev/blog/2016/08/19/right-to-left-support-for-react-native-apps) for more info

**Reply by jcyuan** [reply]
thank you
